src/main/java/sirius/biz/storage/layer2/mongo/MongoVariant.java

```diff
@@ -21,29 +21,60 @@
 import java.time.LocalDateTime;
 import java.util.Optional;
 
+/**
+ * Stores the metadata of a {@link BlobVariant} in the underlying MongoDB.
+ * <p>
+ * Note that all non trivial methods delegate to the associated {@link MongoBlobStorage}.
+ */
 public class MongoVariant extends MongoEntity implements BlobVariant {
 
+    /**
+     * References the raw blob from which this variant was derived.
+     */
     public static final Mapping BLOB = Mapping.named("blob");
     private final MongoRef<MongoBlob> blob = MongoRef.on(MongoBlob.class, BaseEntityRef.OnDelete.CASCADE);
 
+    /**
+     * Contains the name / type of this variant.
+     */
     public static final Mapping VARIANT_NAME = Mapping.named("variantName");
     private String variantName;
 
+    /**
+     * Contains the key of the layer1 object which holds the actual data.
+     * <p>
+     * Note that this remains empty until the conversion has been completed.
+     */
     public static final Mapping PHYSICAL_OBJECT_KEY = Mapping.named("physicalObjectKey");
     private String physicalObjectKey;
 
+    /**
+     * Contains the file size of the converted file.
+     */
     public static final Mapping SIZE = Mapping.named("size");
     private long size = 0;
 
+    /**
+     * Contains the timestamp when the last conversion was attempted.
```

Comment:
Why is this not nullable but the node name on which the conversion was tried on is?